src/main/java/core/basesyntax/Zoo.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Zoo {
+    private AviaryForBirds aviaryForBirds = new AviaryForBirds();
+    private Aviary aviary = new Aviary();
+    private Aquarium aquarium = new Aquarium();
+
+    public static void main(String[] args) {
```

Comment:
It's better to have separate Main class, where programs executes, not to mix it with Zoo logic